fonts/bravura/FONTLOG.txt

```diff
@@ -322,7 +322,7 @@ Changelog
 – Added brass valve trill to 'Brass techniques' range (U+E5EF)
 – Added wind mouthpiece pop and rim only to 'Wind techniques' range (U+E60A, U+E60B)
 – Added bow behind bridge on one, two, three, or four strings to 'String techniques' range (U+E627–U+E62A)
-– Added nasal voice, tongue click, finger click, and tongue and finger click (as used by Stockhausen) to 'Voical techniques' range (U+E648–U+E64A)
+– Added nasal voice, tongue click, finger click, and tongue and finger click (as used by Stockhausen) to 'Vocal techniques' range (U+E648–U+E64A)
```

Comment:
Taken from https://github.com/steinbergmedia/petaluma, so wouzld need to get fixed there (too/first)

Reply:
https://github.com/musescore/MuseScore/pull/11566#discussion_r867539770